src/DripsDeployer.sol

```diff
@@ -0,0 +1,412 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.17;
+
+import {AddressDriver} from "./AddressDriver.sol";
+import {Caller} from "./Caller.sol";
+import {Drips} from "./Drips.sol";
+import {ImmutableSplitsDriver} from "./ImmutableSplitsDriver.sol";
+import {Managed, ManagedProxy} from "./Managed.sol";
+import {NFTDriver} from "./NFTDriver.sol";
+import {OperatorInterface, RepoDriver} from "./RepoDriver.sol";
+import {Ownable2Step} from "openzeppelin-contracts/access/Ownable2Step.sol";
+import {Address} from "openzeppelin-contracts/utils/Address.sol";
+import {Create2} from "openzeppelin-contracts/utils/Create2.sol";
+
+struct Module {
+    bytes32 salt;
+    uint256 amount;
+    bytes initCode;
+}
+
+contract DripsDeployer is Ownable2Step {
+    // slither-disable-next-line naming-convention
+    bytes32[] internal _moduleSalts;
+    address public immutable initialOwner;
+
+    function args() public view returns (bytes memory) {
+        return abi.encode(initialOwner);
+    }
+
+    constructor(address initialOwner_) {
+        // slither-disable-next-line missing-zero-check
+        initialOwner = initialOwner_;
+        _transferOwnership(initialOwner);
+    }
+
+    function deployModules(
```

Comment:
It's 4 lists of modules. Each list must is passed into `cast call` as a separate argument, and Linux by default puts a limit of 128KB per program argument. Because the modules must be hex-encoded, this limits each modules list to 64KB of bytecode. If we did the sane thing and accepted only 1 modules list, we wouldn't be able to do the deployment in a single transaction.